src/util/address.rs

```diff
@@ -166,7 +213,7 @@ impl Privkey {
     /// Converts a private key to an address
```

Comment:
PrivateKey should not be in this module. It should not be an afterthought of address. Compressed/uncompressed is not an attribute of Private key but that of PublicKey. This is not a stopper for this PR but needs clean up.